Tasks/AzureFunctionAppContainerV1/taskparameters.ts

```diff
@@ -1,25 +1,32 @@
 import tl = require('azure-pipelines-task-lib/task');
 
+function convertToNullIfUndefined(arg: any): any {
+    return arg ? arg : null;
+}
+
 export class TaskParametersUtility {
 
     public static async getParameters(): Promise<TaskParameters> {
+
         var taskParameters: TaskParameters = {
             connectedServiceName: tl.getInput('azureSubscription', true),
             ImageName: tl.getInput('imageName', true),
-            AppSettings: tl.getInput('appSettings', false),
-            StartupCommand: tl.getInput('containerCommand', false),
-            ConfigurationSettings: tl.getInput('configurationStrings', false),
+            AppSettings: convertToNullIfUndefined(tl.getInput('appSettings', false)),
+            StartupCommand: convertToNullIfUndefined(tl.getInput('containerCommand', false)),
+            ConfigurationSettings: convertToNullIfUndefined(tl.getInput('configurationStrings', false)),
             WebAppName: tl.getInput('appName', true),
-            ResourceGroupName: tl.getInput('resourceGroupName', false),
-            SlotName:tl.getInput('slotName', false),
-            DeployToSlotOrASEFlag: tl.getBoolInput('deployToSlotOrASE', false),
+            ResourceGroupName: convertToNullIfUndefined(tl.getInput('resourceGroupName', false)),
+            SlotName:convertToNullIfUndefined(tl.getInput('slotName', false)),
+            DeployToSlotOrASEFlag: convertToNullIfUndefined(tl.getBoolInput('deployToSlotOrASE', false)),
```

Comment:
why do we need this?

Reply:
azure-pipelines-task-lib/task is returning `undefined` when the expectation for the code was `null` previously. So changing output here only so we dont have to change the code anywhere else.